packages/core/src/forms/control-label/control-label.element.scss

```diff
@@ -33,7 +33,6 @@
   font-size: $cds-global-typography-secondary-font-size !important;
   font-weight: $cds-global-typography-secondary-font-weight !important;
   color: $cds-global-typography-color-300 !important;
-  line-height: $cds-global-typography-secondary-line-height !important;
```

Comment:
Removing this will cause the label box height to be smaller than the text due to the line height eraser calculation, moving the label closer to the input on vertical style layouts. I think to prevent the layout shift the input could have a 1px padding top to nudge the text down to align with the label.

```scss
// https://github.com/vmware/clarity/blob/next/packages/core/src/input/input.element.scss#L16
:host {
  --padding: #{$cds-global-space-1} #{$cds-global-space-4} 0 #{$cds-global-space-4}; // 1px top for line height alignment with label
}
```

Reply:
Thanks, I needed to bump it to --cds-global-sapce-2 to align with the label text. Image below.